index.html

```diff
@@ -1998,11 +1999,11 @@
           };
         </pre>
         <p>
-          The <a>PaymentRequestUpdateEvent</a> enables the web page to update
+          The <a>PaymentRequestUpdateEvent</a> enables developers to update
           the details of the payment request in response to a user interaction.
         </p>
         <p>
-          If the web page wishes to update the payment request then it should
+          If developers wish to update the payment request then they should
```

Comment:
So there would be inconsistency between _the developer_ and _they_. Is this a convention for gender neutrality?